packages/allure-mocha/package.json

```diff
@@ -29,10 +29,18 @@
     "@types/glob": "^7.1.4",
     "@types/mocha": "^9.0.0",
     "@types/node": "^16.7.8",
+    "@typescript-eslint/eslint-plugin": "^5.31.0",
+    "@typescript-eslint/parser": "^5.31.0",
+    "allure-commandline": "^2.18.1",
```

Comment:
We don't need `allure-commandline` devDependency in every package.
`npm run allure-report --workspace 'packages/allure-mocha'` works like a charm without it

Reply:
👍